dotnet-authserver/tests/TeacherIdentity.AuthServer.Tests/Services/UserImport/UserImportProcessorTests.cs

```diff
@@ -502,7 +502,6 @@ public async Task Process_WithTheoryCsvRowData_InsertsExpectedUserAndEventAndUse
         // Arrange
         using var dbContext = _dbFixture.GetDbContext();
         var userImportStorageService = Mock.Of<IUserImportStorageService>();
```

Comment:
Had a look into removing this mock, but would have to extend HostFixture rather than DbFixture and not sure if it's worth the refactoring